project/plugins.sbt

```diff
@@ -13,4 +13,4 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
 addSbtPlugin("org.scalastyle" % "scalastyle-sbt-plugin" % "1.0.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.4")
 addSbtPlugin("com.47deg"  % "sbt-microsites" % "0.7.20")
-
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.5.10")
```

Comment:
Same q. about scalafix.

Reply:
Yes I will clean up, but I also had to use a local snapshot version of Arrow to use `twitter-util % 18.7.0`

Currently the exception handling is failing: 
- `should give back bad request if we divide by 0 *** FAILED ***`
- `should rescue the exception occurred in it *** FAILED ***`
- `should handle both Error and Errors *** FAILED ***`

Reply:
Need to wrap my head around it, but at least the failures seem consistent

Reply:
Oh interesting. Let's clean up everything else and make sure scalastyle is happy. I can take a look at these failures later this week unless you figure out what's going before that.